template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/s3guard/S3ConfigProviderTest.java

```diff
@@ -29,12 +35,18 @@
 import com.sequenceiq.common.api.type.InstanceGroupType;
 
 @RunWith(MockitoJUnitRunner.class)
-public class S3GuardConfigProviderTest {
-    private S3GuardConfigProvider underTest;
+public class S3ConfigProviderTest {
+
+    @Mock
+    private LocationHelper locationHelper;
+
+    @InjectMocks
+    private S3ConfigProvider underTest;
 
     @Before
     public void setUp() {
-        underTest = new S3GuardConfigProvider();
+        MockitoAnnotations.initMocks(this);
```

Comment:
This call is not necessary when using `@RunWith(MockitoJUnitRunner.class)`. On the other hand, I suggest migrating the class to JUnit 5. 😄

Reply:
Ok, it is now JUnit 5.